packages/app/src/constants/routes.tsx

```diff
@@ -3,11 +3,11 @@ import {
   CasePage,
   CasesPage,
   EditAccountPage,
-  MessagesPage,
-  MessagePage,
+  MessageDetailPage,
```

Comment:
Should we change the names of the other pages as well?

Reply:
I agree.
- CaseDetailPage
- TaskDetailPage

am I missing something?

Reply:
done.